docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -13,70 +22,223 @@ Currently we support GitHub, GitLab, BitBucket, Gitea and Azure DevOps Git servi
 
 Open an [issue](https://github.com/konveyor/pelorus/issues/new) or a pull request to add support for additional Git providers!
```

Comment:
This shouldn't be in development?

Reply:
No, the development could be about *how to write and test new git provider*. Here we are talking from the user perspective, that uses some unsupported git provider, so we tell that user what could be done to make it into supported set.

Reply:
Would be better to put this is getting_started/overview Supported providers section?

Reply:
Moved that there.